docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,24 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Air-gapped registries
```

Comment:
```
#### Allow push of non-distributable artifacts
```

And `--allow-nondistributable-artifacts=myReg1`

No matter what this is going to be confusing since a docker *user* can't mark something as non-dstributable (which would be cool for certain cases) effectively making it a vendor feature, not a user feature.
This is taking something that generally only vendors know about and giving the user a knob to tweak it.